src/servers/src/postgres/server.rs

```diff
@@ -86,7 +86,7 @@ impl PostgresServer {
                                 Some(addr)
                             }
                             Err(e) => {
-                                warn!("Failed to get PostgreSQL client addr, err: {}", e);
+                                warn!("Failed to get PostgreSQL client addr, err: {:?}", e);
```

Comment:
Does e implement ErrorExt?
```suggestion
                                warn!(e; "Failed to get PostgreSQL client addr");
```